docker/manage

```diff
@@ -138,29 +153,16 @@ function echoYellow (){
   echo -e "${_yellow}${_msg}${_nc}" >&2
 }
 
-function copyCustomTheme(){
-  # Look for the theme within the context of the build ...
-  themeFolder=$(find '../client/themes' -type d -name ${THEME})    
-  if [ ! -z "${themeFolder}" ]; then
-    echoYellow "Found the '${THEME}' theme within the build context; '${themeFolder}'"
-  else
-    echoYellow "The '${THEME}' theme was not found within the required build context."
-    echoYellow "Copying the theme into the build context."
-    sourceTheme="${THEME_PATH%/}/${THEME}"
-    destinationTheme="../client/themes/${THEME}"
-    cp -r "${sourceTheme}" "${destinationTheme}"
-  fi
-
-  echo "${destinationTheme}"
+function copyClientRepo(){
+  _client_dirname="../client"
+  _gitignore="../.gitignore"
+  echo "Creating client directory"
+  mkdir -p -- $_client_dirname
+  echo "Pulling client repository from $WEB_REPO_URL"
+  wget -qO- $WEB_REPO_URL | tar -xz - -C $_client_dirname --strip 1
```

Comment:
Assumes `wget`, `curl` would likely be a better assumption as we use it in other scripts.

Reply:
Neither of those will be pre-installed on Windows hosts (I don't think `curl` comes with Git Bash, but I may be wrong). Could implement a check like we do [here](https://github.com/bcgov/issuer-kit/blob/master/docker/manage#L23-L31).

Reply:
I'm wondering if we even need to support copying the client over in a build process. Should we just remove this function altogether?

Reply:
Perhaps not.  Good topic for the upcoming meeting.

Reply:
Yeah, good point. In hindsight, and based on our conversations in the Rocket.Chat channel, it may be best left out of the build and delegated completely to each team/developer - it could be a different framework altogether in which case it wouldn't work anyway.

Reply:
Removed as we are no longer building `web` as part of Aries-VCR